### ISSUE TITLE
Can't get rid of new gradient, new popupLyrics extension doesn't work

### ISSUE THREAD

**Issue by wiktor-waj** [khanhas/spicetify-cli#637]
Hi, I've upgraded spicetify to 2.2.2 and I also have the newest version of spotify avilable on linux. 
But I can't get rid of that new gradient that spotify implemented.
I have just a simple color scheme, no user.css.
It looks like this 
![screenshot](https://imgur.com/AqcG27z.png)
As you can see there is no popupLyrics extension at the top and also this uglyyy gradient that as I can see by looking at screnshots of new versions you managed to get rid of.
Here is my config file (config-xpui.ini):
```
[Setting]
prefs_path              = /home/wiktor/.config/spotify/prefs
color_scheme            = 
inject_css              = 1
spotify_launch_flags    = 
check_spicetify_upgrade = 0
spotify_path            = /opt/spotify
current_theme           = wpgTheme
replace_colors          = 1
overwrite_assets        = 0

[Preprocesses]
disable_sentry        = 1
disable_ui_logging    = 1
remove_rtl_rule       = 1
expose_apis           = 1
disable_upgrade_check = 1

[AdditionalOptions]
extensions  = popupLyrics.js
custom_apps = 

[Patch]

; DO NOT CHANGE!
[Backup]
version = 1.1.56.595.g2d2da0de

```

**Comment by wiktor-waj** [comment]
I should also probably add that gradient problem occurs on SpicetifyDefault theme as well:
![screenshot2](https://imgur.com/uM04wcr.png)

**Comment by khanhas** [comment]
sorry, Spotify Linux v1.1.56 has different implementation of certain things and it does not contain "map" files which I used to translate CSS names. Hence that's why user.css doesn't work and popupLyrics.js extension can't find the element to inject the button in.

**Comment by khanhas** [comment]
If you have a Windows or MacOS machine, you can copy 1.1.58 or 1.1.59 "SPA" files over and run `spicetify backup apply` on them, instead of stock 1.1.56 SPA files

**Comment by khanhas** [comment]
Should be fixed now in spicetify v2.2.3

**Comment by wiktor-waj** [comment]
Everything works and the new lyrics app is awesome. You are absolutely incredible.
